src/page-ui.js

```diff
@@ -0,0 +1,224 @@
+    projectList.insertAdjacentHTML('beforeend', itemContent);
+  });
+}
+
+function addNewProjectForm() {
+  const formSection = document.querySelector('#form-section');
+  const newProjectForm = `
+    <section class="section">
+      <div class="container">
+        <div id="project-form" class="is-hidden">
+
+          <div class="field">
+            <label class="label">Title</label>
+            <div class="control">
+              <input class="input" name="project-name" type="text" placeholder="Project Name">
+            </div>
+          </div>
+
+          <div class="field">
+            <div class="control">
+              <button id="save-project-button" class="button is-success">Save Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>`;
+  formSection.insertAdjacentHTML('beforeend', newProjectForm);
+}
+
+function addNewTodoForm() {
+  const formSection = document.querySelector('#form-section');
+  const newTodoForm = `
+  <section class="section">
+      <div class="container">
+        <div id="todo-form" class="is-hidden">
+
+          <div class="field">
+            <label class="label">Title</label>
+            <div class="control">
+              <input class="input" name="todo-title" type="text" placeholder="Title">
+            </div>
+          </div>
+
+          <div class="field">
+            <label class="label">Description</label>
+            <div class="control">
+              <textarea class="textarea" name="todo-description" placeholder="Description"></textarea>
+            </div>
+          </div>
+
+          <div class="field">
+            <label class="label">Pages</label>
+            <div class="control">
+              <input class="input" name="todo-dueDate" type="date" placeholder="Due date">
+            </div>
+          </div>
+
+          <div class="field">
+            <label class="label">Priority</label>
+            <div class="control">
+              <div class="select">
+                <select name="todo-priority">
+                  <option value="high">High</option>
+                  <option value="normal">Normal</option>
+                  <option value="low">Low</option>
+                </select>
+              </div>
+            </div>
+          </div>
+          
+          <div class="field">
+            <label class="label">Project</label>
+            <div class="control">
+                <div class="select">
+                  <select id="dropdown" name="project-list">
+                  </select>
+                </div>
+            </div>
+          </div>
+
+          <div class="field">
+            <div class="control">
+              <button id="save-todo-button" class="button is-success">Save Todo</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>`;
+  formSection.insertAdjacentHTML('beforeend', newTodoForm);
+}
+
+function generateDropdown() {
```

Comment:
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.